dev-texlive/texlive-latexextra/texlive-latexextra-2023_p69752-r2.ebuild

```diff
@@ -3991,3 +3991,8 @@ TEXLIVE_MODULE_BINSCRIPTS="
 	texmf-dist/scripts/wordcount/wordcount.sh
 	texmf-dist/scripts/yplan/yplan
 "
+
+pkg_postinst() {
+	optfeature_header "Install the following packages for additional features:"
```

Comment:
No need for optfeature_header here:
```suggestion
```

Reply:
Hi @Flowdalic, 
Thanks for the feedback!

I both removed the header and changed the file name to -r3. I had forgotten about it the first time!